test/integration/userStatus.test.js

```diff
@@ -41,11 +40,11 @@ describe("UserStatus", function () {
     await cleanDb();
   });
 
-  describe("GET /user-status", function () {
+  describe("GET /users/status", function () {
```

Comment:
since we are using `/users/` it gives impression that it could be in user.js in test